### PR TITLE
Add Shortcut

### LINKED ISSUE #9327: Change Clubhouse to Shortcut
### Brand Name

Clubhouse

### Official Resources for Icon and Color

It has been renamed to Shortcut https://www.shortcut.com/blog/clubhouses-name-is-now-shortcut
SVG: https://assets-global.website-files.com/6372338e5477e047032b37a5/6372be763e75b42883c24d2c_Shortcut%20Logo.svg

### Additional Comments

This icon needs to be removed to add Clubhouse #6136 

### Contributing

Yes

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
shortcut

**Issue:** closes #9327 

**Similarweb rank:** 54,109. See https://www.similarweb.com/website/shortcut.com

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from website header https://www.shortcut.com
Color: Taken from logo `#58b1e4`

### CODE REVIEW

**Comment by adamrusted** [comment]
As mentioned in the other PR, if you can adapt this to JUST add Shortcut, and keep the existing Clubhouse - this can be merged as a non-breaking change.

**Comment by PratyushJayachandran** [comment]
> As mentioned in the other PR, if you can adapt this to JUST add Shortcut, and keep the existing Clubhouse - this can be merged as a non-breaking change.

Done

File: icons/shortcut.svg
Comment (adamrusted) [inline icons/shortcut.svg:1]: Seeing some [minor differences](https://github.com/simple-icons/simple-icons/assets/1830867/cc5cfc3f-b784-44c5-97f6-f21798277802) between the PR in black, and `Shortcut_mark_blue.svg` from the brand kit in blue. Could you rebuild using that source please?

File: _data/simple-icons.json [hunk 1]
```diff
@@ -11431,6 +11431,12 @@
             "hex": "189EFF",
             "source": "https://www.shopware.com/en/press/press-material/"
         },
+        {
+            "title": "Shortcut",
+            "hex": "58B1E4",
+            "source": "https://www.shortcut.com",
```
Comment (adamrusted) [inline _data/simple-icons.json:11437]:
```suggestion
            "source": "https://www.shortcut.com/branding",
```

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/shortcut.svg
Comment (PratyushJayachandran) [reply icons/shortcut.svg:1]: Re-made from the file you mentioned

Comment (PratyushJayachandran) [reply icons/shortcut.svg:1]: So in this case the website is using a slightly different logo

**Review by adamrusted — APPROVED** [review]
LGTM! Thanks @PratyushJayachandran 🎉

**Pull request merged**